cmd/cli/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/keboola/processor-split-table/internal/pkg/cli"
+	"github.com/keboola/processor-split-table/internal/pkg/log"
+)
+
+func main() {
+	logger := log.NewLogger()
+
+	// Handle panic with correct exit code
+	defer func() {
+		if err := recover(); err != nil {
+			exitWithError(logger, err)
+		}
+	}()
+
+	if err := cli.Run(logger); err != nil {
+		exitWithError(logger, err)
+	}
+}
```

Comment:
Pridal som `main` pre CLI.

Reply:
Processor slicuje tabulky ktore najde v `in`, CLI slicuje iba jednu tabulku, ktora sa zada.